.github/workflows/pr-open.yml

```diff
@@ -22,213 +22,43 @@ jobs:
-          oc_version: "4.13"
-          overwrite: true
-          parameters:
-            -p ZONE=${{ github.event.number }}
-            -p TAG=${{ github.event.number }}
-            -p CHES_TOKEN_URL='https://loginproxy.gov.bc.ca/auth/realms/comsvcauth/protocol/openid-connect/token'
-            -p CHES_API_URL='https://ches.api.gov.bc.ca/api/v1'
-            -p BCREGISTRY_URI='https://bcregistry-prod.apigee.net'
-            -p COGNITO_REGION=ca-central-1
-            -p FRONTEND_URL=${{ needs.vars.outputs.url }}
-
-      - name: Dev data replacement
-        uses: bcgov/action-deployer-openshift@v4.0.0
-        with:
-          file: database/openshift.dev.yml
-          oc_namespace: ${{ secrets.OC_NAMESPACE }}
-          oc_server: ${{ secrets.OC_SERVER }}
-          oc_token: ${{ secrets.OC_TOKEN }}
-          oc_version: "4.13"
-          overwrite: true
-          parameters: -p ZONE=${{ github.event.number }}
-
-      - name: Deploy Frontend ConfigMap
-        uses: bcgov/action-deployer-openshift@v4.0.0
-        with:
-          file: frontend/openshift.configmap.dev.yml
-          oc_namespace: ${{ secrets.OC_NAMESPACE }}
-          oc_server: ${{ secrets.OC_SERVER }}
-          oc_token: ${{ secrets.OC_TOKEN }}
-          oc_version: "4.13"
-          overwrite: true          
-          parameters:
-            -p ZONE=${{ github.event.number }}  
-
-      - name: Deploy Frontend
-        uses: bcgov/action-deployer-openshift@v4.0.0
-        with:
-          file: frontend/openshift.deploy.yml
-          oc_namespace: ${{ secrets.OC_NAMESPACE }}
-          oc_server: ${{ secrets.OC_SERVER }}
-          oc_token: ${{ secrets.OC_TOKEN }}
-          oc_version: "4.13"
-          overwrite: true
-          parameters:
-            -p ZONE=${{ github.event.number }}
-            -p TAG=${{ github.event.number }}
-            -p VITE_NODE_ENV=openshift-dev
-            -p URL=${{ needs.vars.outputs.url }}
-            -p GREEN_DOMAIN=${{ secrets.GREEN_DOMAIN }}
-            -p COGNITO_REGION=${{ secrets.COGNITO_REGION }}
-            -p COGNITO_CLIENT_ID=${{ secrets.COGNITO_CLIENT_ID }}
-            -p COGNITO_USER_POOL=${{ secrets.COGNITO_USER_POOL }}
-            -p COGNITO_DOMAIN=${{ secrets.COGNITO_DOMAIN }}
-            -p COGNITO_ENVIRONMENT=DEV
-            -p LANDING_URL=${{ needs.vars.outputs.url }}
-
-  tests:
-    name: Tests
-    needs: [deploy, vars]
-    uses: ./.github/workflows/.automated-tests.yml
+    name: Deploys (${{ github.event.number }})
+    # needs: [builds]
     secrets: inherit
-    with:
-      url: ${{ needs.vars.outputs.url }}
-
-  results:
-    name: PR Results
-    needs: [builds, deploy, tests]
-    if: always()
-    runs-on: ubuntu-24.04
-    steps:
-      - if: contains(needs.*.result, 'failure')||contains(needs.*.result, 'canceled')
-        run: echo "At least one job has failed." && exit 1
-      - run: echo "Success!"
+    uses: ./.github/workflows/.deploy.yml
+
+  # tests:
```

Comment:
Here too, maybe it's better to uncomment this block here as well before you forget.